components/eamxx/src/control/atmosphere_surface_coupling_exporter.cpp

```diff
@@ -268,6 +357,7 @@ void SurfaceCouplingExporter::do_export(const double dt, const bool called_durin
     PF::calculate_z_mid(team, num_levs, z_int_i, z_mid_i);
```

Comment:
This chunk of code above is only needed if you need to export `Sa_z` and/or `Sa_tbot`, so you could do something like
```
bool compute_z = h_export_source(idx_Sa_tbot)==EAMXX || h_export_source(idx_Sa_z)==EAMXX);
parfor (..) {
  if (compute_z) {
    const Real z_surf = 0.0;
    PF::calculate_z_int(team, num_levs, dz_i, z_surf, z_int_i);
    ...
   }
};
```